great-idea/services.html

```diff
@@ -1,107 +1,93 @@
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
-
-
-
-iOS Development
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
-
-
-
-Android Development
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
-
-
-Some Facts About Our Services
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-
-Awesome thing
-Amazing thing
-Cool thing
-Great thing
-
-<img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
-
-Copyright Great Idea! 2018
-
-</body>
-
+    <head>
+    <meta charset="utf-8">
+    <title>Great Idea!</title>
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web|Fira Mono" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+
+    <!--[if lt IE 9]>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+    </head>
+    <body>
+        <div class="top-overlay"></div>
+        <div class="container">
+            <header>
+                <nav>
+                    <a class="main-link" href="#">Services</a>
+                    <a class="main-link" href="#">Product</a>
+                    <a class="main-link" href="#">Vision</a>
+                    <a class="main-link" href="#">Features</a>
+                    <a class="main-link" href="#">About</a>
+                    <a class="main-link" href="#">Contact</a>
+                    <a class="logo-link" title="Great Idea! Company logo." href="index.html"></a>
+                </nav>
+                <img class="header-image" src="http://i63.tinypic.com/u9d38.jpg" alt="Our services header image">
+                <section class="header-description">
+                    <h1>Services</h1>
+                    <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+                </section>
+            </header>
+            <div class="services">
+                <article>
```

Comment:
Great use of article tags! It shows you understand the difference between an article and a section.